README.md

```diff
@@ -119,301 +111,238 @@ docker-compose up
 
 Configuration
 -------------
-
-Change your default database connection name in `config/database.php`:
-
-```php
-'default' => env('DB_CONNECTION', 'mongodb'),
-```
-
-And add a new mongodb connection:
+You can use MongoDB either as the main database, either as a side database. To do so, add a new `mongodb` connection to `config/database.php`:
 
 ```php
 'mongodb' => [
-    'driver'   => 'mongodb',
-    'host'     => env('DB_HOST', 'localhost'),
-    'port'     => env('DB_PORT', 27017),
-    'database' => env('DB_DATABASE'),
-    'username' => env('DB_USERNAME'),
-    'password' => env('DB_PASSWORD'),
-    'options'  => [
-        'database' => 'admin' // sets the authentication database required by mongo 3
-    ]
+    'driver' => 'mongodb',
+    'host' => env('DB_HOST', '127.0.0.1'),
+    'port' => env('DB_PORT', 27017),
+    'database' => env('DB_DATABASE', 'homestead'),
+    'username' => env('DB_USERNAME', 'homestead'),
+    'password' => env('DB_PASSWORD', 'secret'),
+    'options' => [
+        'database' => 'admin', // required with Mongo 3+
```

Comment:
For some providers can be different from admin. 
For example using Heroku auth admin is equal to env('DB_DATABASE')

Reply:
On some projects i use it as

```php
'database' => env('DB_AUTHENTICATION_DATABASE', 'admin'),
```

Do this sound good?

Reply:
Perfect!